code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -131,12 +131,15 @@
 	. = new /list
 	if(air)
 		var/list/gases = air.gases
-		for(var/id in gases)
-			var/gas = gases[id]
-			var/gas_meta = gas[GAS_META]
-			var/gas_overlay = gas_meta[META_GAS_OVERLAY]
-			if(gas_overlay && gas[MOLES] > gas_meta[META_GAS_MOLES_VISIBLE])
-				. += gas_overlay
+		if(gases)
```

Comment:
has this specific line been demonstrated to improve performance? I'm not sure it will

Reply:
I tested it once and saw the numbers slightly lower with it, i'll try a run with a counter to test if I can remember

Reply:
Colour me unconvinced, the for loop will either do the exact same check itself or do it faster